torch/csrc/autograd/python_variable_indexing.cpp

```diff
@@ -334,39 +276,20 @@ PyObject* THPVariable_getitem(PyObject* self, PyObject* index) {
   }
 
   // indexing by tensors ("advanced" indexing)
-  return wrap(dispatch_index(sliced, variableIndices));
+  return wrap(([&]() {
```

Comment:
I saw all the calls to `wrap` were deleted in the C++ version but not here. Aren't the calls to wrap redundant in both cases now?

Reply:
I believe they are not redundant, because `wrap` takes a Tensor and returns a `PyObject`, and we need to return a `PyObject` from the `THPVariable_getitem` function.